extensions/amp-analytics/analytics-vendors-list.md

```diff
@@ -455,7 +467,11 @@ Type attribute value: `piStats`
 
 Type attribute value: `pressboard`
 
-Adds support for Pressboard. Configuration details can be found at [help.pressboard.ca] `http://help.pressboard.ca/publisher-resources/getting-started/implementing-google-amp`.
+<!--
+(Link 404s, unclear whether vendor is still supported.)
```

Comment:
@omar84

Hi! It looks like you originally added this analytics config on #18169

The original documentation link 404s, and it's unclear whether AMP is still supported by this vendor.

- Is the vendor integration still active?
- If so, is there a newer link?

Thanks.

Reply:
Hi @alanorozco 
I'm with Pressboard (the vendor) and taking over from @omar84. We're still actively supporting AMP. Any assistance on how I can get started would be greatly appreciated. I'll setup a new documentation link asap.

Reply:
@tiamkorki Awesome, good to know. The implementation is still in place, not sure if it requires any later work. Please ping me or @micajuine-ho [on Slack](https://github.com/ampproject/amphtml/blob/main/CONTRIBUTING.md#discussion-channels) if you need specific assistance.